src/index.template.html

```diff
@@ -249,9 +192,16 @@
             window.location.reload();
           }
         }
-      //If in 5 seconds it hasn't render the root of the page thne reload the page.
+      //If in 5 seconds it hasn't render the root of the page then reload the page.
       setTimeout(forceRefreshIfNotloaded, 30000);
+
+      window.addEventListener('load', function (event) {
+        if (!('noModule' in HTMLScriptElement.prototype)) {
+          document.getElementById('error').innerHTML =
+            "<b>⚠️ Error!</b> The version of this browser doesn't support some essential features used in InterPro.<br/><br/>Please upgrade your browser.";
```

Comment:
```suggestion
            "Sorry, your web browser does not support JavaScript modules, which are required for InterPro. Please upgrade your browser to a newer version, or switch to a different browser that supports JavaScript modules.";
```

Reply:
Well, maybe the error message is not required since there is a toast...

Reply:
It's actually needed for the extreme case where none of the website JS gets loaded.